Podfile

```diff
@@ -43,7 +43,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-    pod 'WordPressKit', '~> 4.21.0'
+    pod 'WordPressKit', '~> 4.22-beta.2'
```

Comment:
We leave off the beta version so we don't have to update it _every_ time. So this should just be `4.22-beta`.

Reply:
That makes too much sense. 😆.